serverless.yml

```diff
@@ -94,6 +86,10 @@ resources:
 #      Description: The URI that webhooks should be sent to
 #      Value: !GetAtt WebhookHttpApi.ApiEndpoint
   Resources:
+    WebhookDeadLetterQueue:
```

Comment:
@mnapoli this is actually fairly simple! Still a lot less code overall than the manual webhook handling via a queue. And actually a smaller file overall than using the `queue` component in lift